.travis.yml

```diff
@@ -15,11 +15,10 @@ env:
   - CHEF_VERSION=12.4.1
   - CHEF_VERSION=12.3.0
   - CHEF_VERSION=12.2.1
-  - CHEF_VERSION=12.1.2
+  - CHEF_VERSION=12.1.2    # known broken due to ffi-yajl versioning issues.
```

Comment:
Note: I am still testing, but I think you can avoid the rvm / chef_version cross by doing something like this: https://github.com/chef/ruby-project-template/blob/master/.travis.yml#L20

Basically, use `include:` in the Matrix to include CHEF_VERSION ./ rvm pairs, and don't bother pairing different CHEF_VERSIONs with different RVMs. "Matrix" becomes "test latest released chef against Ruby 2.0 and Ruby 2.1", and "test master and 12.1-12.3 against Ruby 2.1." Eliminates Ruby 2.0 + master/12.1-12.3 combos.